src/ResultsActivity/ResultsActivity.js

```diff
@@ -117,6 +139,66 @@ class ResultsActivity extends Component {
       </div>
     )
 
+    const unpaidOffenders = new Set()
+
+    const attendAllNonAssigned = async () => {
+      try {
+        this.setState({ payingAll: true })
+
```

Comment:
This would be better as a reducer so it can be tested separately.

Reply:
Not sure a reducer is necessary for setting one piece of local state.

Reply:
It's more for consistency and testability really, definitely more of a code style point here.

Reply:
I think a reducer for this would be overkill, Nev.  It's just local state for this component rather than global state shared across components.

Reply:
Happy with that :)